src/core/types/verifier_detail.rs

```diff
@@ -55,4 +58,75 @@ impl VerifierDetail {
             entity_detail,
         }
     }
+
+    pub fn to_v2(self) -> VerifierDetailV2 {
+        let total_fee_cost = self.onboarding_cost.mul(self.fee_percent);
+        VerifierDetailV2 {
+            address: self.address,
+            onboarding_cost: self.onboarding_cost,
+            onboarding_denom: self.onboarding_denom,
+            fee_amount: total_fee_cost,
```

Comment:
question: why do we have the fee amount here vs. just calculating it from the list of fee destinations, or just letting it be implicit based on their sum? Seems somewhat redundant... Could still check that the sum is less than the total onboarding cost (leaving the amount the verifier receives implicit, as it effectively is now)

Reply:
Totally agreed.  I removed the top-level `VerifierDetailV2.fee_amount` field and the logic is much simpler to work with, now.  Great idea